### PR TITLE
Feature/geode 3175

### PR DESCRIPTION
**Pull request by bschuchardt**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ x] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by bschuchardt** [comment]
@kohlmu-pivotal @hiteshk25 @galen-pivotal @pivotal-amurmann

**Comment by pivotal-amurmann** [comment]
Should this be squashed?

File: geode-core/src/test/java/org/apache/geode/test/dunit/standalone/ChildVM.java [hunk 1]
```diff
@@ -49,8 +49,8 @@ public static void main(String[] args) throws Throwable {
       int pid = OSProcess.getId();
       logger.info("VM" + vmNum + " is launching" + (pid > 0 ? " with PID " + pid : ""));
       if (!VersionManager.isCurrentVersion(geodeVersion)) {
-        logger.info("This VM is using Geode version {}. Version.CURRENT is {}", geodeVersion,
-            Version.CURRENT);
+        logger.info("This VM is using Geode version {}. Version.CURRENT is {} (ordinal={})",
```
Comment (galen-pivotal) [inline geode-core/src/test/java/org/apache/geode/test/dunit/standalone/ChildVM.java]: Worth adding the old ordinal here too?

File: geode-core/src/test/java/org/apache/geode/test/dunit/standalone/ProcessManager.java [hunk 2]
```diff
@@ -90,12 +92,17 @@ public synchronized void launchVM(String version, int vmNum, boolean bouncedVM)
...
+      Process process = Runtime.getRuntime().exec(cmd, null, workingDir);
+      pendingVMs++;
+      ProcessHolder holder = new ProcessHolder(process);
+      processes.put(vmNum, holder);
+      linkStreams(version, vmNum, holder, process.getErrorStream(), System.err);
+      linkStreams(version, vmNum, holder, process.getInputStream(), System.out);
+    } catch (RuntimeException | Error t) {
+      t.printStackTrace();
+      throw t;
+    }
```
Comment (galen-pivotal) [inline geode-core/src/test/java/org/apache/geode/test/dunit/standalone/ProcessManager.java]: Worth logging a message telling us where it is printed from with the stack trace?

**Pull request closed without merging**